README.md

```diff
@@ -22,13 +22,15 @@ The Programmer's Handbook is an open source initiative to collect, archive, and
 
 If you'd like to contribute to the project, please fork it, add your article, and submit a PR for review!
 
------------------------------------------------------------------------
```

Comment:
This is just `Prettier` doing it's thing. Helps to keep everything formatted similarly.